torchci/lib/bot/utils.ts

```diff
@@ -13,7 +13,7 @@ export function isPyTorchOrg(owner: string): boolean {
 }
 
 export function isPyTorchPyTorch(owner: string, repo: string): boolean {
-  return isPyTorchOrg(owner) && repo === "pytorch";
+  return isPyTorchOrg(owner) && ["pytorch", "vision"].includes(repo);
 }
```

Comment:
This is wrong, please add new function, say `isPyTorchCoreOrTorchvision` or `isPyTorchVision`

Reply:
Alternatively we can just have a Dr. CI specific one.

`isDrCIEnabled`

Reply:
I like the idea of having `isDrCIEnabled` 👍